src/R.sln

```diff
@@ -130,6 +130,9 @@ EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Microsoft.R.Containers.Windows.Test", "Windows\Containers\Test\Microsoft.R.Containers.Windows.Test.csproj", "{62E1CA8E-8398-46FC-B14F-DE0C5759DD2D}"
 EndProject
 Global
+	GlobalSection(Performance) = preSolution
+		HasPerformanceSessions = true

```

Comment:
?

Reply:
Oh boy, VS adds this when you use profiler... Its harmless, but lemme try and undo it.